cmd/eudico/mir.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/lotus/eudico-core/global"
+)
+
+var mirCmd = &cli.Command{
+	Name:  "mir",
+	Usage: "Mir consensus",
+	Subcommands: []*cli.Command{
+		daemonCmd(global.MirConsensus),
+	},
+}
```

Comment:
Let's leave it for the next PR, but I think we discussed already, it would be great if we can integrate the `mir-validator` as part of Eudico also. I will track it on an issue.